scripts/changelog.js

```diff
@@ -51,14 +51,15 @@ const formatChanges = (changes) =>
         .replace(/[a-z]+\((\w+)\)/, '$1')
         // Linkify commit refs.
         .replace(
-          /^(\w{8})/,
+          /^(\w+)/,
           '* [[$1](https://github.com/boardgameio/boardgame.io/commit/$1)]'
         )
         // Linkify PR references.
         .replace(
           /\(#(\d{3,})\)/,
           '([#$1](https://github.com/boardgameio/boardgame.io/pull/$1))'
         )
+        .replace(/\)](\w*)/, ')]$1')
```

Comment:
It's not always super easy to understand what a regex is intended to do. What's the intention of this addition?

Reply:
Oops, you're right 😅  that line is no longer necessary as we no longer need to re-introduce the space between `[commidID]` and `commit message`

Reply:
Ahhh, ok! Gotcha 😄